dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java

```diff
@@ -602,4 +590,18 @@ private Path getSelectedLocalPath(final TreeTableRow<FileTreeModel> row) {
         }
         return localPath;
     }
+
+    private static class TableCell extends TreeTableCell<FileTreeModel, Number> {
```

Comment:
This looks almost identical to `TreeTableCell` in file `JobInfoPresenter.java` can we get rid of one of them?